README.md

```diff
@@ -10,7 +10,7 @@ Building
 make all
 ```
 
-The project uses essentially a copy and paste of the Makefile from the [regular input plugin for mupen64plus](https://github.com/mupen64plus/mupen64plus-input-sdl) With the dynamic config lib and SDL dependencies removed.
+The project uses essentially a copy and paste of the Makefile from the [regular input plugin for mupen64plus](https://github.com/mupen64plus/mupen64plus-input-sdl) With the dynamic config lib and SDL dependencies removed. It includes flags to link libjson-c, which it assumes has been installed in /usr/local/lib and /usr/local/include (you can get this library [here](https://github.com/json-c/json-c)).
```

Comment:
TBH, I didn't even look for a pre-built package in the package manager. I found it on GitHub and just pulled it down and built it. I'll take a look at the package and see if I can simplify this a bit. Thanks for the tip.